libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1845,6 +1850,22 @@ bool RC_Channel::do_aux_function(const AUX_FUNC ch_option, const AuxSwitchPos ch
     }
 #endif
 
+#if AP_EXTERNAL_CONTROL_ENABLED
+    case AUX_FUNC::EXTERNAL_CONTROL: {
+        AP_ExternalControl *external_control = AP_ExternalControl::get_singleton();
+        if (external_control != nullptr) {
+            if (ch_flag == AuxSwitchPos::HIGH) {
+                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "RC: External Control Enabled");
```

Comment:
Put these in `RC_Channel::LookupTable RC_Channel::lookuptable` instead?

Reply:
I like that.